### PR TITLE
[14.0][UPD] pos_cash_move_reason: chatter record when money in or out.

### PR DESCRIPTION
**Pull request by ilyasProgrammer**

### CODE REVIEW

File: pos_cash_move_reason/wizard/wizard_pos_move_reason.py [hunk 1]
```diff
@@ -90,6 +90,18 @@ def apply(self):
         self.ensure_one()
         AccountBankStatementLine = self.env["account.bank.statement.line"]
         AccountBankStatementLine.create(self._prepare_statement_line())
+        if self.move_type == "income":
+            chatter_text = "Money in.\n"
```
Comment (legalsylvain) [inline pos_cash_move_reason/wizard/wizard_pos_move_reason.py]:
```suggestion
            chatter_text = _("Money in.\n")
```

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Nice improvment, thanks ! 

Could you make all the text translatable ? 

thanks !

File: pos_cash_move_reason/wizard/wizard_pos_move_reason.py (hunk 1 above)
Comment (ilyasProgrammer) [reply pos_cash_move_reason/wizard/wizard_pos_move_reason.py]: sure

**Review by francesco-ooops — APPROVED** [review]
LGTM!

@legalsylvain what do you say?

File: pos_cash_move_reason/wizard/wizard_pos_move_reason.py [hunk 2]
```diff
@@ -90,6 +90,20 @@ def apply(self):
...
+        if self.move_type == "income":
+            chatter_text = _("<p>Money in.</p>")
+        elif self.move_type == "expense":
+            chatter_text = _("<p>Money out.</p>")
+        chatter_text += _(
+            "<p>Move reason: %s</p><p>Reason: %s</p> <p>Amount: %s</p>"
+        ) % (
+            self.move_reason_id.name,
+            self.name,
+            self.amount,
```
Comment (legalsylvain) [inline pos_cash_move_reason/wizard/wizard_pos_move_reason.py]: Nitpicking / not blocking : Could you format amount ? 

image

There is a function ``format_lang``  here : https://github.com/odoo/odoo/blob/14.0/odoo/tools/misc.py#L1290
Something like that could do the job.


```suggestion
            formatLang(self.env, self.amount, currency_obj=self.journal_id.currency_id),
```

**Review by legalsylvain — APPROVED** [review]
Hi @ilyasProgrammer. Thanks for the change ! 

A nice to have / non blocking point online.

thanks !

File: pos_cash_move_reason/wizard/wizard_pos_move_reason.py (hunk 2 above)
Comment (ilyasProgrammer) [reply pos_cash_move_reason/wizard/wizard_pos_move_reason.py]: ok

**Comment by legalsylvain** [comment]
thanks ! 
/ocabot merge patch

**Pull request merged**